src/selectBetweenOffsets.ts

```diff
@@ -0,0 +1,83 @@
+import * as vscode from "vscode";
+import { ExtensionHostMessageHandler, MessageType } from "../shared/protocol";
+
+const addressRe = /^0x[a-f0-9]+$/i;
+const decimalRe = /^[0-9]+$/i;
+
+export const showSelectBetweenOffsets = (messaging: ExtensionHostMessageHandler): void => {
+    const startingInput = vscode.window.createInputBox();
+    const endingInput = vscode.window.createInputBox();
+    startingInput.placeholder = "Enter offset to select from";
+    endingInput.placeholder = "Enter offset to select until";
+
+    messaging.sendEvent({ type: MessageType.StashDisplayedOffset });
+
+    let fromOffset: number | undefined;
+    let toOffset: number | undefined;
+    let accepted = false;
+
+    startingInput.onDidChangeValue(value => {
```

Comment:
It seems we duplicate a lot of logic between the starting and ending inputs, can it be pulled into a helper function?

Reply:
Yes, working on it

Reply:
How is it now?